### PR TITLE
feat: add provider parameter TDE-623

### PR DESCRIPTION
**Pull request by paulfouquet**
https://github.com/linz/topo-imagery/pull/348

### CODE REVIEW

File: workflows/imagery/standardising.yaml [hunk 1]
```diff
@@ -48,6 +48,117 @@ spec:
...
+            "Sinergise Ltd",
+            "South Taranaki District Council",
+            "South Waikato District Council",
+            "South Wairarapa District Council",
+            "Southland District Council",
+            "Stratford District Council",
+            "Taranaki Regional Council",
+            "Tararua District Council",
+            "Tasman District Council",
+            "Taupo District Council",
```
Comment (dwsilk) [inline workflows/imagery/standardising.yaml:137]:
```suggestion
            "Taupō District Council",
```

File: workflows/imagery/standardising.yaml [hunk 2]
```diff
@@ -48,6 +48,117 @@ spec:
         value: "*Region/District/City* *GSD* *Urban/Rural* Aerial Photos (*Year-Year*)"
       - name: description
         value: "Orthophotography within the *Region Name* region captured in the *Year*-*Year* flying season."
+      - name: producer
+        value: "AAM NZ"
```
Comment (dwsilk) [inline workflows/imagery/standardising.yaml:52]:
```suggestion
        value: "Unknown"
```
Let's add an Unknown to the enum and make that the default

File: workflows/imagery/standardising.yaml [hunk 3]
```diff
@@ -48,6 +48,117 @@ spec:
...
+            "Beca",
+            "GeoSmart",
+            "Landpro",
+            "Maxar",
+            "NZ Aerial Mapping",
+            "Terralink International",
+            "UAV Mapping NZ",
+          ]
+      - name: licensor
+        value: "Ashburton District Council"
```
Comment (dwsilk) [inline workflows/imagery/standardising.yaml:66]: Ditto with an Unknown here too

**Review by dwsilk — CHANGES_REQUESTED** [review]

File: workflows/imagery/standardising.yaml [hunk 4]
```diff
@@ -334,7 +449,6 @@ spec:
         resources:
           requests:
             cpu: 3000m
-            memory: 7.8Gi
```
Comment (amfage) [inline workflows/imagery/standardising.yaml:337]: The memory request should be retained.

Comment (paulfouquet) [reply workflows/imagery/standardising.yaml:337]: merging issue!

File: workflows/imagery/standardising.yaml [hunk 5]
```diff
@@ -48,6 +48,119 @@ spec:
...
+        enum:
+          [
+            "Unknown",
+            "AAM NZ",
+            "Aerial Surveys",
+            "Beca",
+            "GeoSmart",
+            "Landpro",
+            "Maxar",
+            "NZ Aerial Mapping",
```
Comment (rclarkelinz) [inline workflows/imagery/standardising.yaml:62]:
```suggestion
            "NZ Aerial Mapping",
            "Recon Limited",
```

File: workflows/imagery/standardising.yaml [hunk 6]
```diff
@@ -48,6 +48,119 @@ spec:
...
+            "Lower Hutt City Council",
+            "Mackenzie District Council",
+            "Manawatū District Council",
+            "Manawatū-Whanganui LASS Ltd",
+            "Marlborough District Council",
+            "Masterton District Council",
+            "Matamata-Piako District Council",
+            "Maxar Technologies",
+            "Ministry of Primary Industries",
+            "NZ Aerial Mapping Ltd",
```
Comment (dwsilk) [inline workflows/imagery/standardising.yaml:115]:
```suggestion
            "NZ Aerial Mapping",
```

File: workflows/imagery/standardising.yaml [hunk 7]
```diff
@@ -48,6 +48,119 @@ spec:
...
+            "Ōtorohanga District Council",
+            "Otago Regional Council",
+            "Palmerston North City Council",
+            "Porirua City Council",
+            "Queenstown-Lakes District Council",
+            "Rangitīkei District Council",
+            "Rotorua District Council",
+            "Ruapehu District Council",
+            "Selwyn District Council",
+            "Sinergise Ltd",
```
Comment (dwsilk) [inline workflows/imagery/standardising.yaml:130]:
```suggestion
            "Sinergise",
```

File: workflows/imagery/standardising.yaml [hunk 8]
```diff
@@ -48,6 +48,119 @@ spec:
...
+            "South Waikato District Council",
+            "South Wairarapa District Council",
+            "Southland District Council",
+            "Stratford District Council",
+            "Taranaki Regional Council",
+            "Tararua District Council",
+            "Tasman District Council",
+            "Taupō District Council",
+            "Tauranga City Council",
+            "Terralink International Ltd",
```
Comment (dwsilk) [inline workflows/imagery/standardising.yaml:141]:
```suggestion
            "Terralink International",
```

**Review by dwsilk — CHANGES_REQUESTED** [review]

**Review by rclarkelinz — APPROVED** [review]

**Review by dwsilk — APPROVED** [review]

**Review by amfage — APPROVED** [review]

**Pull request merged**